deploy/main.py

```diff
@@ -75,10 +75,10 @@
 # Content Security Policy
 #
 
-# Google Cloud Storage
-securityheaders.addCspRule("img-src", "storage.googleapis.com", "enforce")
+securityheaders.addCspRule("connect-src", "self", "enforce")  # fixme: Temporary fix, will be enforced by viur-core soon
```

Comment:
https://github.com/viur-framework/viur-core/pull/315 is merged, so you can remove this.